connection.go

```diff
@@ -50,8 +52,8 @@ type auth struct {
 
 func (ws *Ws) connect() (err error) {
 	d := websocket.Dialer{
-		WriteBufferSize:  8192,
-		ReadBufferSize:   8192,
+		WriteBufferSize:  ws.writeBufSize,
```

Comment:
is there a way to have them still default to 8192?  Otherwise I think this is good.

Reply:
Thanks, added defaults of 8192